integrations/README.md

```diff
@@ -5,3 +5,5 @@ This folder contains the integrations for OpenDAL. Integrations are used to inte
 ## Available Integrations
 
 - [`object_store_opendal`](./object_store): Use OpenDAL as a backend for the [object_store](https://docs.rs/object_store/latest/object_store/).
+
+- [`dav-server-fs-opendal`](./dav-server-fs-opendal/): Use OpenDAL as a backend to access data in various service with WebDAV protocol.
```

Comment:
This crate's name is `dav-server-opendalfs` (follow the upstream pattern), but the folder is `dav-server` (since it's in our project).